tests/rt_utils.sh

```diff
@@ -360,13 +360,18 @@ check_results() {
             printf ".......ALT CHECK.."
             if [[ ${MACHINE_ID} =~ orion || ${MACHINE_ID} =~ hera || ${MACHINE_ID} =~ gaea || ${MACHINE_ID} =~ jet || ${MACHINE_ID} =~ cheyenne ]] ; then
               nccmp -d -f -g -B --Attribute=checksum --warn=format ${RTPWD}/${CNTL_DIR}/${i} ${RUNDIR}/${i} > ${i}_nccmp.log 2>&1 && d=$? || d=$?
+              if [$d -ne 0] && [ $d -ne 1 ]; then
```

Comment:
This is not correct syntax. I am getting this error:
```
 Comparing RESTART/20210322.090000.fv_tracer.res.tile1.nc ............ALT CHECK..rt_utils.sh: line 363: [1: command not found        
....NOT OK                                                                                                                           
 Comparing RESTART/20210322.090000.fv_tracer.res.tile2.nc ............ALT CHECK..rt_utils.sh: line 363: [1: command not found        
....NOT OK                                                                                                                           
```
Did you test this in your branch?  Please rewrite it as:
```
if [[ $d -ne 0 && $d -ne 1 ]]; then
```

Reply:
updated!